pkg/satellite/service.go

```diff
@@ -77,13 +77,17 @@ func (s *Service) CreateUser(ctx context.Context, userInfo UserInfo, companyInfo
 
 // CreateCompany creates Company for authorized User
 func (s *Service) CreateCompany(ctx context.Context, info CompanyInfo) (*Company, error) {
-	user, err := s.Authorize(ctx)
-	if err != nil {
-		return nil, err
+	claims, ok := GetClaims(ctx)
+	if !ok {
+		return nil, errs.New("unauthorized")
```

Comment:
Let's create a specific error class `ErrUnauthorized` so that it can be checked elsewhere.